samples/fabric-healthcare-data-solutions/sample-notebooks/hds_alm_helper.ipynb

```diff
@@ -0,0 +1,1336 @@
+      "source": [
+        "updated_config, replacements = get_updated_deployment_parameters_configuration(source_workspace_id, source_admin_lakehouse_id, dest_workspace_id, dest_admin_lakehouse_id, solution_name)\n",
+        "\n",
+        "print(\"Here are the artifacts that were udpated with their destination ids:\\n\")\n",
+        "\n",
+        "for replaced_artifact, replaced_artifact_id in replacements.items():\n",
+        "    print(f\"{replaced_artifact}: {replaced_artifact_id}\")\n",
+        "\n",
+        "print(\"\\n\")\n",
+        "print(\"This is the updated deployment parameters configuration for the destination workspace:\\n\")\n",
+        "print(json.dumps(updated_config, indent=2))"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "6dd7d0f8-7af7-4832-acea-a5a12eef340a",
+      "metadata": {
+        "microsoft": {
+          "language": "python",
+          "language_group": "synapse_pyspark"
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "#### Save the deployment parameters to the destination workspace\n",
+        "\n",
+        "After reviewing the updated deployment parameters configuration, run the following cell to persist the updated configuration to the destination workspace. Please not this will override the existing `deploymentParametersConfiguration.json` file if it already exists in that location."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "f1b17615-fed4-47ed-900b-740fecd6980a",
+      "metadata": {
+        "editable": false,
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "microsoft": {
+          "language": "python",
+          "language_group": "synapse_pyspark"
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        },
+        "run_control": {
+          "frozen": false
+        }
+      },
+      "outputs": [],
+      "source": [
+        "from os import path\n",
+        "system_configuration_source_path = get_system_configurations_path(source_workspace_id, source_admin_lakehouse_id)\n",
+        "system_configuration_destination_path = path.dirname(get_system_configurations_path(dest_workspace_id, dest_admin_lakehouse_id))\n",
+        "deployment_parameters_destination_path = get_deployment_parameters_destination_path(dest_workspace_id, dest_admin_lakehouse_id)\n",
+        "\n",
+        "mssparkutils.fs.cp(system_configuration_source_path, system_configuration_destination_path, recurse=True)\n",
+        "mssparkutils.fs.put(file = deployment_parameters_destination_path, content = json.dumps(updated_config), overwrite=True)\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "f5e8672d",
+      "metadata": {},
+      "source": [
+        "#### Copy System Data\n",
```

Comment:
Should we wrap these function calls in flags and set the flags at the top of the notebook all to true? Make it easy for the customer to run a subset or skip certain steps on subsequent runs?